assets/app/scripts/directives/oscFormSection.js

```diff
@@ -0,0 +1,24 @@
+"use strict";
+
+angular.module("openshiftConsole")
+  .directive("oscFormSection", function () {
+    return {
+      restrict: "E",
+      transclude: true,
+      scope: {
+        header: "@",
+        about: "@",
+        aboutTitle: "@",
+        editText: "@",
+        expand: "@"
+      },
+      templateUrl: "views/directives/osc-form-section.html",
+      link: function(scope, element, attrs){
+        if(!attrs.editText) attrs.editText="Edit";
```

Comment:
@jwforres here we are simply trying to default a value if they are not defined.  I believe they are always undefined on the scope